cmd/controller-manager/app/controllermanager.go

```diff
@@ -66,8 +66,10 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 	opts := options.NewOptions()
 
 	cmd := &cobra.Command{
-		Use:  "karmada-controller-manager",
-		Long: `The karmada controller manager runs a bunch of controllers`,
+		Use: "karmada-controller-manager",
+		Long: `The karmada-controller-manager runs the various controllers.
+The controllers watch Karmada objects and then talk to the underlying clusters' API servers 
```

Comment:
It's not  only karmada objects, also like deployment in karmada control plane?

Reply:
Mainly karmada objects. Refer to README.md.